index.html

```diff
@@ -36,5 +36,151 @@ <h5 class="connect">LET'S CONNECT</h5>
         </section>
       </main>
     </div>
+
+    <div>
+      <section>
```

Comment:
- [ ] Kindly try not to wrap your `section` tag inside a `div` tag to follow HTML best practices and semantics.you got this 👍